servers/dsub/jobs/test/base_test_cases.py

```diff
@@ -64,6 +72,13 @@ def create_app(self):
             app = connexion.App(__name__, specification_dir='../swagger/')
             app.app.json_encoder = JSONEncoder
             app.add_api('swagger.yaml')
+
+            # TEST_TOKEN_VALIE is a global config value used to distinguish testing jobs from batch to batch by timestemp
+            app.app.config['TEST_TOKEN_VALUE'] = self.test_token_label[
+                'test_token']
+            # AGGREGATION_JOB_NAME_FILTER is a global config value used to distinguish aggregation testing jobs from others
+            app.app.config[
+                'AGGREGATION_JOB_NAME_FILTER'] = "aggregation-testing-unique"
```

Comment:
Per offline, switch to just check test token

switch naming to be `config['AGGREGATION_JOB_LABEL_FILTER'] = 'test_token=...'`